.travis.yml

```diff
@@ -64,7 +56,6 @@ matrix:
     - name: "stability (Chrome Dev)"
       if: type = pull_request
       os: linux
-      sudo: required
```

Comment:
Surely this requires sudo to write `/etc/hosts`?

Reply:
But Chrome needs `/etc/hosts` which `hosts_fixup` will try to write to, no? What description says it's redundant? (I see it manages to write it, so I don't know what description you're referring to here.)

Reply:
I mean the description of this PR. That doesn't say *why* it's redundant, I left that implied: because it's already at the top-level scope of `.travis.yml`.